hatch/src/repo.rs

```diff
@@ -65,11 +65,11 @@ pub fn clone_project_deps(path: &Path,
 }
 
 fn clone_nested_project_deps(path: &Path, visited: &mut HashSet<String>) -> HatchResult<bool> {
-  let current_project = yaml::parse(hatchfile_path(path).as_path())?;
+  let current_project = read_project(path)?;
```

Comment:
In the code where I have been using the tasks, I have been only importing the tasks module and using the syntax: `task::read_project(path)` to give a better context about what read_project is.